tools/python/maps_generator/README.md

```diff
@@ -8,292 +8,96 @@ maps built by a generator_tool newer than the app.**
 ## What are maps?
 
 Maps are .mwm binary files with special meta-information for rendering, searching, routing, and other use cases.
-Files from [data/borders](https://github.com/organicmaps/organicmaps/tree/master/data/borders) define map boundaries for each individual file. The world is segmented into separate files by these boundaries, with the intent of having managebly small files to download. These files are referred to as *maps* or *countries*. A country is referring to one of these files, not necesarily a geographic country. Also note that there are two special countries called *World* and *WorldCoasts*. These are small simplified maps of the world and coastlines used when other maps have not yet been downloaded.
+Files from [data/borders](https://github.com/organicmaps/organicmaps/tree/master/data/borders) define map boundaries for each individual file. The world is segmented into separate files by these boundaries, with the intent of having managebly small files to download. These files are referred to as *maps* or *countries*. A country is referring to one of these files, not necesarily a geographic country. Also note that there are two special countries called *World* and *WorldCoasts*. These are small simplified maps of the world and coastlines (sea and ocean watercover) used when other maps have not yet been downloaded.
 
 ## Setup
 
-You must have Python version >= 3.6 and complete the following steps:
+You must have Python version >= 3.7 and complete the following steps:
```

Comment:
Do we use some 3.7 features?

Reply:
its because of 3bbc4a15bf00988b935ba06f4a2107c526377b77
the `requests` module requires python 3.7 now